test/integration_tests/with_db/udfs/test_sequence_classification_single_text_script.py

```diff
@@ -15,44 +15,48 @@ def test_sequence_classification_single_text_script(
     n_labels = 3  # negative, neutral, positive
 
     n_rows = 100
-    input_data = []
-    for i in range(n_rows):
-        input_data.append(
-            (
-                "",
-                bucketfs_conn_name,
-                str(model_params.sub_dir),
-                model_params.sequence_class_model_specs.model_name,
-                "I am so happy to be working on the Transformers Extension.",
+    for return_ranks, n_result_per_input in [("ALL", n_labels), ("HIGHEST", 1)]:
+        # this test does not check for the mixed return ranks case, are we fine with that?
+        input_data = []
+        for i in range(n_rows):
```

Comment:
Why can't you use the list comprehension?

Reply:
it just is like this in all the tests. if we want to change it, i would like to change it everywhere, but not in this pr